File: Python/섹션 2 (완) - 코드 구현력 기르기/9. 주사위 게임 (중)/AA.py
import sys
#sys.stdin=open('input.txt', 'rt')

# 개수 n 입력 받기
n=int(input())

result=list()
for i in range(n):
    # 입력 3개 리스트에 받기
    arr=list(map(int, input().split()))

    cnt=0
    temp=0
    if arr[0]==arr[1]:
        cnt+=1
        if arr[1]==arr[2]:
            cnt+=1
        else:
            if arr[0]==arr[2]:
                cnt+=1
    else:
        if arr[1]==arr[2]:
            cnt+=1
            temp=arr[1]
        if arr[0]==arr[2]:
            cnt+=1
            temp=arr[0]

                
    if cnt==2:
        result.append(10000+arr[0]*1000)
    elif cnt==1:
        result.append(1000+temp*100)
    elif cnt==0:
        result.append(100*max(arr))
    else:
        print("에러 발생!")

print(max(result))

File: Javascript/섹션2. 1-2차원 배열 탐색/5. 등수구하기 (중)/AA.js
function solution(num, arr){
    
    let result=[]
    let cnt=0
    for(let i=0;i<num;i++){
        cnt=0
        now=arr[i]
        for(let j=0;j<num;j++){
            if(now==arr[j]) continue
            if(now<arr[j]){
                cnt++
            }
        }
        result.push(cnt+1) // 등수니까 +1
    }

    return result
}
let arr=[87, 89, 92, 100, 76];
console.log(solution(5, arr))